integration-tests/src/test/java/tachyon/master/MasterFaultToleranceIntegrationTest.java

```diff
@@ -115,6 +116,35 @@ public void faultTest() throws IOException {
   }
 
   @Test
+  public void deleteFileFaultTest() throws Exception {
+    // Kill leader -> create files -> kill leader -> delete files, repeat.
+    List<Pair<Long, TachyonURI>> answer = Lists.newArrayList();
+    for (int kills = 0; kills < MASTERS - 1; kills ++) {
+      Assert.assertTrue(mLocalTachyonClusterMultiMaster.killLeader());
+      CommonUtils.sleepMs(Constants.SECOND_MS * 2);
+
+      if (kills % 2 != 0) {
```

Comment:
Both parts of the branch should have a comment on what it will be doing. (one is deleting files, the other is creating files?)